connectors/migrations/20240702_gdrive_fullsync_pptx_docx.ts

```diff
@@ -0,0 +1,36 @@
+import { launchGoogleDriveFullSyncWorkflow } from "@connectors/connectors/google_drive/temporal/client";
+import logger from "@connectors/logger/logger";
+import { ConnectorModel } from "@connectors/resources/storage/models/connector_model";
+
+const { LIVE } = process.env;
+
+// Syncing all pptx and docx files for all existing Google Drive connectors.
+export async function main() {
+  const connectors = await ConnectorModel.findAll({
+    where: {
+      type: "google_drive",
+      errorType: null,
+      pausedAt: null,
+    },
+  });
+  const additionalFilter =
```

Comment:
Wouldn't it be less error prone for the futures to call that `filterMimeTypes`, this would be an array of the mime types we want to filter, and then you construct the string by looping on this array?

Reply:
You even have a mimeTypesToSync that we could reuse for that